app/serializers/site_status_serializer.rb

```diff
@@ -26,12 +26,9 @@ def status
     object.status_before_type_cast
   end
 
-  # rubocop:disable Style/DateTime
   def course_open_date
-    # TODO applications_accepted_from should be a timestamp, not a string
-    DateTime.parse(object.applications_accepted_from).iso8601 if object.applications_accepted_from.present?
+    object.applications_accepted_from.iso8601 if object.applications_accepted_from.present?
```

Comment:
Do we need to convert to `iso8601`? I thought the data is stored in the correct format for us now?

Reply:
You're right. I don't think we need this especially as the data for this field has no time element.